core/trino-spi/src/main/java/io/trino/spi/security/AccessDeniedException.java

```diff
@@ -171,11 +174,19 @@ public static void denyRenameSchema(String schemaName, String newSchemaName, Str
         throw new AccessDeniedException(format("Cannot rename schema from %s to %s%s", schemaName, newSchemaName, formatExtraInfo(extraInfo)));
     }
 
+    /**
+     * @deprecated Use {@link #denySetEntityAuthorization(String, List, TrinoPrincipal)}
+     */
+    @Deprecated
```

Comment:
All of the deprecations should be marked with `forRemoval=true`

Reply:
`forRemoval = true` added